test/clj/rems/test_browser.clj

```diff
@@ -1521,7 +1595,7 @@
       (btu/wait-page-loaded)
       (select-option "Handlers" "handler")
       (select-option "Handlers" "carl")
-      (select-option "Forms" "Simple form")
+      (select-option "Forms" "Private field form")
```

Comment:
Hmm why are we replacing the "Simple form" with "Private field form"?

Reply:
i assumed that `Simple form` didn't have any use so i hijacked it into my specific use case 🙈  but in this case it turned out that there was one test case using it. maybe this could use a small discussion on what would be best course of action

Reply:
I guess in general a "Simple form" is a useful demo/test form to have there. And we can add some more for this specific case and more specific demo/test uses?

Reply:
this change was reverted, demo data on public/private fields is now in test data